doc/install/main.rst

```diff
@@ -0,0 +1,167 @@
+Manage Qserv deployment
+-----------------------
+
+Prerequisites
+~~~~~~~~~~~~~
+
+For all setups
+^^^^^^^^^^^^^^
+
+-  Access to a Kubernetes v1.14.2+ cluster via a valid ``KUBECONFIG`` file.
```

Comment:
Consider making a reStructuredText substitution for the Kubernetes version requirement since this appears in one other place.